topics/mvvmcross/index.md

```diff
@@ -0,0 +1,24 @@
+---
+aliases: monocross
+created_by: Stuart Lodge
+display_name: MvvmCross
+github_url: https://github.com/mvvmcross
+logo: mvvmcross.png
+related: xamarin, dotnet, framework
+released: June 2013
+short_description: The .NET MVVM framework for cross-platform solutions, including Xamarin.iOS, Xamarin.Android, Windows, and Mac.
+topic: mvvmcross
+url: https://www.mvvmcross.com/
+---
+MvvmCross is a cross-platform model-view-view model framework that enables developers to create powerful cross platform mobile applications. It supports Xamarin.iOS, Xamarin.Android, Xamarin.Mac, Xamarin.Forms, Universal Windows Platform (UWP), and Windows Presentation Framework (WPF).
+
+The high level features that MvvmCross provides you with are:  
+
+   • MVVM architecture pattern  
```

Comment:
I think the bulleted list should be omitted or rephrased as a sentence that lists just a few of the bullet points, comma-separated.

Reply:
Changes applied. Thank you for suggestions!